packages/kbn-i18n/tsconfig.browser.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.browser.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "incremental": true,
+    "outDir": "./target/web",
+    "declaration": false,
+    "isolatedModules": true,
+    "sourceMap": true,
+    "sourceRoot": "../../../../../packages/kbn-i18n/src",
+    "types": [
+      "node"
```

Comment:
do we need `node` types for browser build?

Reply:
👍 we don't